CODE_OF_CONDUCT.md

```diff
@@ -2,7 +2,7 @@
 
 As contributors and maintainers of this project, we pledge to respect all people who contribute through reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, and other activities.
 
-We are committed to making participation in this project a harassment-free experience for everyone, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, age, or religion.
+We are committed to making participation in this project a harassment-free experience for everyone, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, or religion.
```

Comment:
Not sure if you're trolling here. But to clarify what I meant: My suggestion is meant to cover hateful speech like "Republicans are evil", "The Social Justice movement is evil."

Just like people shouldn't bring up one's race, gender, etc. when discussing their code, people shouldn't bring up one's political ideology, and certainly not harass them on any of those aspects. Open source projects should welcome people regardless of their race, gender, political ideology, etc.